implementations/rust/ockam/ockam_command/src/node/models/show.rs

```diff
@@ -0,0 +1,163 @@
+use std::fmt::Display;
+
+use colorful::Colorful;
+
+use ockam_multiaddr::{
+    proto::{DnsAddr, Node, Tcp},
+    MultiAddr,
+};
+use serde::Serialize;
+
+use crate::output::Output;
+
+use super::{
+    portal::{ShowInletStatus, ShowOutletStatus},
+    secure_channel::ShowSecureChannelListener,
+    services::ShowServiceStatus,
+    transport::ShowTransportStatus,
+};
+
+/// Information to display in the `ockam node show` command
+#[derive(Debug, Serialize)]
+pub struct ShowNodeResponse<'a> {
```

Comment:
I think you could get rid of that lifetime. It's only used by one of the fields, so it's not doing much. Removing it will also simplify the definition of the traits it implements.

Reply:
Removed